client-tez/src/main/java/org/apache/tez/common/TezIdHelper.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.tez.common;
+
+
+import org.apache.uniffle.common.util.IdHelper;
+
+public class TezIdHelper implements IdHelper {
+  @Override
+  public long getTaskAttemptId(long blockId) {
+    return RssTezUtils.getTaskAttemptId(blockId);
+  }
+
+  public static void main(String[] args) throws Exception {
```

Comment:
Could we convert this method `main` to unit test?

Reply:
Ok, we correct it and push the code

Reply:
We corrected it and pushed to:
https://github.com/apache/incubator-uniffle/pull/890/files#diff-d8ab9d3e5eba99e8e60426b6ed721e336f8377d5099d585738ff7c725648d72a